dev/release/release-config.jsonc

```
@@ -4,11 +4,11 @@
     "slackAnnounceChannel": "announce-engineering"
   },
   "scheduledReleases": {
-    "4.5.0": {
-      "codeFreezeDate": "2023-02-15T10:00:00.000-08:00",
-      "securityApprovalDate": "2023-02-15T10:00:00.000-08:00",
-      "releaseDate": "2023-02-22T10:00:00.000-08:00",
-      "current": "4.5.0",
+    "4.6.0": {
+      "codeFreezeDate": "2023-03-15T10:00:00.000-07:00",
```

This should be 13th.
```suggestion
      "codeFreezeDate": "2023-03-13T10:00:00.000-07:00",
```

[Slack thread](https://sourcegraph.slack.com/archives/C032Z79NZQC/p1673906059971619)

---

TIL, thanks